src/ds/d400/d400-options.cpp

```diff
@@ -2,7 +2,7 @@
 // Copyright(c) 2016 Intel Corporation. All Rights Reserved.
 
 
-#include "ds/d400/d400-thermal-monitor.h"
+#include "ds/ds-thermal-monitor.h"
```

Comment:
Is this include needed here?

Reply:
I just changed all references to the moved code, but your are right, this is not needed. Removing.